MicrosoftTeams.schema.json

```diff
@@ -203,13 +256,26 @@
                                 "personal"
                             ]
                         }
+                    },
+                    "platforms": {
+                        "type": "array",
+                        "description": "Specifies the platform on which these tabs can be shown",
+                        "maxItems": 3,
+                        "items": {
+                            "enum": [
+                                "web",
+                                "desktop",
+                                "mobile"
+                            ]
+                        }
                     }
                 },
                 "required": [
                     "entityId",
                     "name",
                     "contentUrl",
-                    "scopes"
+                    "scopes",
+                    "platforms"
```

Comment:
should we add platforms to required? We could keep all platforms as default and specific platforms can be disabled overriding this property. 
For new manifests, now everyone will need to add this extra property which in most cases people want to show everywhere

Reply:
this will be applied to newer manifest versions, if am not wrong. So who ever is upgrading their manifest definition will add this field. It would be better if developer declares this field explicitly. Right now we are not sure wether we want to make it enabled for all platforms by default or not as most of the developers wouldn't have considered mobile as a surface for their tabs. So it will be kind of a guess work for us.